src/routes/error-handler.ts

```diff
@@ -8,7 +8,7 @@ export const errorHandler: ErrorRequestHandler = (err: any, req: Request, res: R
 
   switch (err.status) {
     case 400:
-      logger.error(err, `400 error detected for ${req.originalUrl}: ${message}`);
+      logger.warn(err, `400 error detected for ${req.originalUrl}: ${message}`);
```

Comment:
400s probably don't need to spam the logs with errors, as this is usually a validation fail and not an actual application error.